.ci/jobs.yml

```diff
@@ -1,26 +1,24 @@
 JOB:
-  - kibana-intake
```

Comment:
You will want to revert the changes in this file to allow all the tests to run as normal.